src/sentry/static/sentry/app/views/eventsV2/results.tsx

```diff
@@ -299,10 +368,35 @@ class Results extends React.Component<Props, State> {
                   setError={this.setError}
                   onChangeShowTags={this.handleChangeShowTags}
                   showTags={showTags}
+                  confirmedQuery={confirmedQuery}
                 />
               </StyledMain>
               {showTags ? this.renderTagsTable() : null}
             </ContentBox>
+            <Confirm
+              priority="primary"
+              header={<strong>{t('May lead to thumb twiddling')}</strong>}
+              confirmText={t('Do it')}
+              cancelText={t('Nevermind')}
+              onConfirm={this.longQueryConfirmed}
+              onCancel={this.longQueryCancelled}
+              message={
+                <p>
+                  {tct(
+                    `You've created a query that will search for events made
+                    [dayLimit] for [projectLimit].
+                    A lot has happened during that time, so this might take awhile.
+                    Are you sure you want to do this?`,
+                    {
+                      dayLimit: <strong>{t('over more than 30 days')}</strong>,
+                      projectLimit: <strong>{t('more than 10 projects')}</strong>,
+                    }
+                  )}
```

Comment:
This can be simplified to 
```suggestion
                  {tct(
                    `You've created a query that will search for events made
                    [dayLimit:over more than 30 days] for [projectLimit:more than 10 projects].
                    A lot has happened during that time, so this might take awhile.
                    Are you sure you want to do this?`,
                    {
                      dayLimit: <strong/>,
                      projectLimit: <strong/>,
                    }
                  )}
```

This way, when we extract the string for translators, they see the entire sentence, instead of breaking out `dayLimit` and `projectLimit` into separate strings.